sechub-scan/src/main/java/com/daimler/sechub/domain/scan/report/HTMLReportCSSFragementGenerator.java

```diff
@@ -0,0 +1,58 @@
+package com.daimler.sechub.domain.scan.report;
+
+import java.io.File;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+
+/**
+ * How to use ? Why this generator?
+ * 
+ * 1. As a web designer I want to change my CSS files and style the report
+ * without always restarting the server + fetching new reports etc.
+ * 
+ * 2. Change the CSS styles - how to ? Enable the web developer mode - see
+ * HTMLScanResultReportModelBuilder.java and start the server. Generate some
+ * testdata and download a HTML report.
+ * 
+ * 3. The report does not look well, because the css file cannot be loaded (same
+ * origin policy...)
+ * 
+ * 4. store the report as local html file and load again css does now apply and
+ * you can design...
+ * 
+ * 5. After css file is as wanted... --> Start this generator. --> will adopt
+ * css data into fragement file.
+ * 
+ * 6. commit push done...
+ * 
+ * @author Albert Tregnaghi
+ *
+ */
+public class HTMLReportCSSFragementGenerator {
```

Comment:
It is nice to have a generator to keep the files in sync. What about moving this into the developer tools, because it is not production code?

Reply:
Ok. Done.